packages/react-table/src/components/Table/CollapseColumn.tsx

```diff
@@ -11,19 +11,21 @@ export interface CollapseColumnProps {
   onToggle?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
   isOpen?: boolean;
   'aria-label'?: string;
+  variant?: 'compact';
 }
 
 export const CollapseColumn: React.FunctionComponent<CollapseColumnProps> = ({
   className = '' as string,
   children = null as React.ReactNode,
   isOpen,
   onToggle,
+  variant,
   ...props
 }: CollapseColumnProps) => (
   <Fragment>
     {isOpen !== undefined && (
       <Button
-        className={css(className, isOpen && styles.modifiers.expanded)}
+        className={css(className, isOpen && styles.modifiers.expanded, variant === 'compact' && 'pf-m-small')}
```

Comment:
Can you update this to conditionally apply `size="sm"` instead of adding the class manually?